CHANGELOG.md

```diff
@@ -4,6 +4,81 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.9.0] - 2022-06-14
+
+### Added
+
+- Cluster scope install mode [#713](https://github.com/3scale/3scale-operator/pull/713)
+- Tenant CRD: delete tenant [#715](https://github.com/3scale/3scale-operator/pull/715)
+- APIManager CRD: reconcile apicast environment vars [#720](https://github.com/3scale/3scale-operator/pull/720)
+- Tenant CRD: store access token instead of the provider key [#725](https://github.com/3scale/3scale-operator/pull/725)
+- APIManager CRD: upgrade mysql to 8 [#682](https://github.com/3scale/3scale-operator/pull/682)
+- Product CRD: delete product [#723](https://github.com/3scale/3scale-operator/pull/723)
+- Backend CRD: delete backend [#723](https://github.com/3scale/3scale-operator/pull/723)
+- Tenant ownership to products and backends [#727](https://github.com/3scale/3scale-operator/pull/727)
+- APIManager CRD: external components optional [#733](https://github.com/3scale/3scale-operator/pull/733) [#737](https://github.com/3scale/3scale-operator/pull/737)
+
+### Changed
+
+- APIManager CRD: remove AMP_RELEASE from system env var set [#654](https://github.com/3scale/3scale-operator/pull/654)
+- APIManager CRD: Cleanup Redis message bus references [#686](https://github.com/3scale/3scale-operator/pull/686)
+- APIManager CRD: New metering labels [#679](https://github.com/3scale/3scale-operator/pull/679)
```

Comment:
would it make sense to add a note here that HA fields will be "converted" to external components = true?

Reply:
Cool, thanks Eguzki